src/main/java/org/broadinstitute/dsm/util/ElasticSearchUtil.java

```diff
@@ -224,12 +224,40 @@ public static ElasticSearch getParticipantESDataByParticipantId(@NonNull String
         return elasticSearch;
     }
 
+    public static ElasticSearch getParticipantESDataByAltpid(@NonNull String index, @NonNull String altpid) {
```

Comment:
Looks like just duplication of `fetchESDataByParticipantId` since `fetchESDataByParticipantId` method determines which to use `altpid` or `guid` as a filter according to `participantId`.

Reply:
This one only uses altpid. fetchESDataByParticipantId wouldn't be relevant in this case. The same staff those methods have is in separate method